frontend/packages/process-editor/src/components/ConfigPanel/ConfigPanel.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import classes from './ConfigPanel.module.css';
+import { VersionAlert } from './VersionAlert';
+import { supportsProcessEditor } from '../../utils/processEditorUtils';
+import { useTranslation } from 'react-i18next';
+import { Alert, Heading, Paragraph } from '@digdir/design-system-react';
+
+export interface ConfigPanelProps {
+  appLibVersion: string;
+}
+
+/**
+ * @component
+ *  Displays the configuration panel area of the ProcessEditor
+ *
+ * @property {string}[appLibVersion] - The app-lib version the user has
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const ConfigPanel = ({ appLibVersion }: ConfigPanelProps) => {
+  const { t } = useTranslation();
+  const isEditAllowed: boolean = supportsProcessEditor(appLibVersion);
```

Comment:
Here is an example of how a centralized `appLibVersion` check could be used.
```suggestion
  const { t } = useTranslation();
  const { isEditAllowed } = useBpmnContext();
```